src/zeek.bif

```diff
@@ -922,6 +922,41 @@ function paraglob_equals%(p_one: opaque of paraglob, p_two: opaque of paraglob%)
 	);
 	%}
 
+%%{
+enum FNV_HASH_TYPE { FNV1a32, FNV1a64};
+
+uint64_t fnva(zeek::Val* input, uint64_t offset, uint64_t prime, FNV_HASH_TYPE type ) {
+	zeek::ODesc desc(zeek::DESC_READABLE);
```

Comment:
I'm forth and back suggesting to revert this to DESC_BINARY so the type is included in the hash. If someone want's READABLE, they can do the following explicitly now `fnv1a32(cat(val))`.

```
assert fnv1a32(123) == fnv1a32("123")
assert fnv1a32(123.0) == fnv1a32("123.0")
assert fnv1a32(T) == fnv1a32("T")
```

Reply:
I have made the changes as you suggested and added testcases for the int/double/string 123 versions to check they have different values.